pages/golfers/[id].js

```diff
@@ -0,0 +1,67 @@
+import Layout from '../../components/Layout'
+import ScorePostWidget from '../../components/ScorePostWidget'
+import ScoreCard from '../../components/ScoreCard'
+import { useEffect, useState } from 'react'
+import { getToken, getUsername } from '../../lib/userAuth'
+import { useRouter } from 'next/router'
+import useUserScores from '../../lib/useUserScores'
+import { USER_SCORES_URL } from '../../lib/useUserScores'
+import { useUserName } from '../../lib/useUserName'
+import { USER_URL } from '../../lib/useUserName'
+
+
+
+const Profile = () => {
+  const router = useRouter()
+  const { id } = router.query
+  // const [ scores, setScores ] = useState([])
+  // const [ scores, error ] = useState('')
```

Comment:
When adding comments to a commit, determine if they could help other people understand the code nearby or if they might be needed in the future.